pkg/users/account-create-dialog.js

```diff
@@ -349,64 +427,90 @@
         }
 
         const footer = {
             actions: [
                 {
                     caption: _("Create"),
                     style: "primary",
-                    clicked: () => {
-                        return passwd_check(
-                            false, // force weak password was NOT clicked
-                            false, // force user with non-unique UID was NOT clicked
-                            state.real_name,
-                            state.user_name,
-                            state.password,
-                            state.password_confirm,
-                            state.locked,
-                            state.change_passw_force,
-                            state.uid,
-                            accounts,
-                            state.min_uid,
-                            state.max_uid,
-                            change
-                        );
-                    },
-                    disabled: state.confirm_weak || state.uid_exists
+                    clicked: () => passwd_check(
+                        false, // force weak password was NOT clicked
+                        false, // force user with existing home directory was NOT clicked
+                        false, // force user with non-unique UID was NOT clicked
+                        state.real_name,
+                        state.user_name,
+                        state.password,
+                        state.password_confirm,
+                        state.locked,
+                        state.home_dir,
+                        state.change_passw_force,
+                        state.uid,
+                        accounts,
+                        state.min_uid,
+                        state.max_uid,
+                        change
+                    ),
+                    disabled: state.confirm_weak || state.uid_exists || state.home_dir_exists
                 }
             ]
         };
+        if (state.home_dir_exists) {
+            footer.actions.push(
+                {
+                    caption: _("Create and change ownership of home directory"),
+                    style: "warning",
+                    clicked: () => passwd_check(
+                        false, // force weak password was NOT clicked
+                        true, // force user with existing home directory was WAS clicked
+                        false, // force user with non-unique UID was NOT clicked
+                        state.real_name,
+                        state.user_name,
+                        state.password,
+                        state.password_confirm,
+                        state.locked,
+                        state.home_dir,
+                        state.change_passw_force,
+                        state.uid,
+                        accounts,
+                        state.min_uid,
+                        state.max_uid,
+                        change
```

Comment:
These 18 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230324-000238-1c01f86d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)